tests/opentracing_instrumentation/test_postgres.py

```diff
@@ -18,10 +18,10 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-from builtins import object
```

Comment:
is there some linter that tells you to remove this? can we add it to `make lint`?

Reply:
> is there some linter that tells you to remove this?

No, I just found that this import is not really required here.